readthedocs/doc_builder/director.py

```diff
@@ -458,18 +460,28 @@ def run_build_commands(self):
             # Execute ``asdf reshim python`` if the user is installing a
             # package since the package may contain an executable
             # See https://github.com/readthedocs/readthedocs.org/pull/9150#discussion_r882849790
-            for reshim_command in reshim_commands:
+            for reshim_command in python_reshim_commands:
                 # Convert tuple/list into set to check reshim command is a
                 # subset of the command itself. This is to find ``pip install``
                 # but also ``pip -v install`` and ``python -m pip install``
-                if reshim_command.issubset(command.split()):
+                if python_reshim_command.issubset(command.split()):
```

Comment:
The variable names are incorrect in these blocks. `python_reshim_command` is not defined.

Reply:
Heh yea, 🙃